web/src/app/schedules/ScheduleAssignedToList.js

```diff
@@ -18,22 +18,12 @@ const query = gql`
   }
 `
 
-export default class ScheduleAssignedToList extends React.PureComponent {
-  static propTypes = {
+export default function ScheduleAssignedToList(props) {
+  ScheduleAssignedToList.propTypes = {
     scheduleID: p.string.isRequired,
   }
```

Comment:
This portion will need to be moved to the bottom of the file, just outside of the component now that it is a function instead of a class